frontend/src/components/ObjetoPerdidoForm.js

```diff
@@ -76,10 +76,10 @@ class ObjetoPerdidoForm extends React.Component{
               <Input type="text" name="lugar" placeholder="lugar" value={this.state.objetoPerdido.lugar} onChange={this.handleChange}/>
             </FormGroup>
             <FormGroup>
-              <Label for="estado">Perdido/Encontrado</Label>
-              <Input type="select" name="select" value={this.state.objetoPerdido.estado} onChange={this.handleChange}>
-                <option>true</option>
-                <option>false</option>                
+              <Label for="Estado">Perdido/Encontrado</Label>
```

Comment:
Tenés que cambiar también el contenido del `Label`. El atributo `for` ponelo en minúscula, porque tiene que corresponder con el `name` del `Input`.